app/views/users/index-old.html.erb

```diff
@@ -0,0 +1,101 @@
+<h1>Users</h1>
```

Comment:
I wonder if we need to hold on to the old users index page if we have the new one in React. Thoughts?